airbyte-integrations/connectors/source-salesforce/acceptance-test-config.yml

```diff
@@ -31,6 +31,8 @@ acceptance_tests:
             bypass_reason: "impossible to fill the stream with data because it is an organic traffic"
           - name: "ActiveScratchOrgShare"
             bypass_reason: "impossible to fill the stream with data because it is an organic traffic"
+          - name: "Describe"
+            bypass_reason: "impossible to fill the stream with data because it is an organic traffic"
```

Comment:
I believe this one is not a valid reason, please fill-in the valid one.

Reply:
updated